controllers/storagecluster/prometheus/localcephrules.yaml

```
@@ -11,10 +11,10 @@ spec:
   - name: ceph.rules
     rules:
     - expr: |
-        kube_node_status_condition{condition="Ready",job="kube-state-metrics",status="true"} * on (node) group_right() max by (node, namespace) (label_replace(ceph_disk_occupation{job="rook-ceph-mgr"},"node","$1","exported_instance","(.*)"))
+        kube_node_status_condition{condition="Ready",job="kube-state-metrics",status="true"} * on (node) group_right() max by (node, namespace) (label_replace(ceph_disk_occupation,"node","$1","exported_instance","(.*)"))
```

```suggestion
        kube_node_status_condition{condition="Ready",job="kube-state-metrics",status="true"} * on (node) group_right() max by (node, namespace) (label_replace(ceph_disk_occupation{job="rook-ceph-exporter"},"node","$1","exported_instance","(.*)"))
```
This is better than completely dropping the job. Let's use it in all places.
We want to add a job filter as we don't want to consider similar metrics from other sources.

---

Made the changes in all the places